builder/ebs/builder_acc_test.go

```diff
@@ -1283,14 +1283,10 @@ func TestAccBuilder_AssociatePublicIPWithoutSubnet(t *testing.T) {
 						return fmt.Errorf("did not change the public IP setting for the instance")
 					}
 
-					if !strings.Contains(string(logs), "No VPC ID provided, Packer will choose one from the provided or default VPC") {
+					if !strings.Contains(string(logs), "No VPC ID provided, Packer will use the default VPC") {
```

Comment:
nit: It might be worth adding the same acceptance test to the other builders

Reply:
Agreed it can't hurt, that being said, since the step is common, all the builders that import it will have the same behaviour.
And in terms of acceptance testing right now, besides `ebs`, the only builder that has the infra to run them is `ebssurrogate`, which essentially works the same as `ebs`, so I think for now we can consider this good to go.
Independently from this PR, we should add acceptance tests for the other builders at some point.